### DISCUSSION TITLE
Digest Auth

### DISCUSSION THREAD

**Discussion by eduardofr275** [viniciussanchez/RESTRequest4Delphi#138] [General]
Alguém já consegui autenticar o Restrequest com Digest?? alguma Dica?

**Comment by schmitzmarcos** [comment]
Olá, também estou precisando disso, mas ainda não consegui fazer funcionar.

**Comment by ZeCarlinhos** [comment]
Pessoal, qualquer ajuda é bem vinda, pois tenho exatamente este problema.
